tests/ctests/CMakeLists.txt

```diff
@@ -1,10 +1,19 @@
 cmake_minimum_required(VERSION 3.10)
 
 # collect all of the test sources
-file(GLOB test_source "${CMAKE_SOURCE_DIR}/tests/ctests/*.cpp")
+set (ALE_TEST_SOURCE ${CMAKE_SOURCE_DIR}/tests/ctests/IsdTests.cpp
+                     ${CMAKE_SOURCE_DIR}/tests/ctests/OrientationsTests.cpp
+                     ${CMAKE_SOURCE_DIR}/tests/ctests/RotationTests.cpp
+                     ${CMAKE_SOURCE_DIR}/tests/ctests/StatesTests.cpp
+                     ${CMAKE_SOURCE_DIR}/tests/ctests/TestInterpUtils.cpp
+                     ${CMAKE_SOURCE_DIR}/tests/ctests/TestMain.cpp)
+
+if(ALE_BUILD_LOAD)
+  list(APPEND ALE_TEST_SOURCE ${CMAKE_SOURCE_DIR}/tests/ctests/LoadTests.cpp)
```

Comment:
Because we only want to include the tests for load(s) if we build them, we can't just glob everything.